src/node-resolver.ts

```diff
@@ -24,7 +24,7 @@ type NodeResolverOptions = {
 
 export function createNodeResolver({
   extensions = ['.mjs', '.cjs', '.js', '.json', '.node'],
-  conditionNames = ['default', 'module', 'import', 'require'],
+  conditionNames = ['import', 'require', 'default'],
```

Comment:
`module` is "faux" ESM, a.k.a. some old bundler implementing old ESM conditions might read this field.

Should we kip this or not though?

Reply:
Maybe you mean `module` in `mainFields`?

Reply:
Maybe you mean `module` in `mainFields`?

Reply:
> Maybe you mean `module` in `mainFields`?

Hmmmmm. I don't remember where I saw `module` in the export map. But if I don't remember, that is probably not important.

Reply:
`module` is most likely only being adopted by the bundler. Node.js never adopts it.

webpack uses `['browser', 'module', 'main']` for apps targeted web and `['module', 'main']` for apps targeted node.

`@rollup/plugin-node-resolve` uses `['module', 'main']`.

I would prefer `['module', 'main']`, where the `module` field would guaranteed to be ESM (where `main` could be either CJS or ESM) and ESM works well with `eslint-plugin-import-x`'s (and `eslint-plugin-import`'s) built-in AST analyzer.